api/conf/conf.go

```diff
@@ -36,7 +36,7 @@ const (
 	EnvDEV   = "dev"
 	EnvLOCAL = "local"
 
-	WebDir = "./html"
+	WebDir = "/html"
```

Comment:
It seems wrong @nic-chen

Reply:
yes, it should be `html/`

Reply:
@membphis  @nic-chen  Done, please have a look~